pages/declaration.js

```diff
@@ -45,20 +45,24 @@ class Declaration extends React.Component {
           ref="submitRegistration"
           csrfToken={this.props.csrfToken}
         >
-          <BackButton {...this.props} />
+          <BackButton {...this.props} t={this.props.t} />
           <ProcessedErrorSummary
+            t={this.props.t}
```

Comment:
ProcessedErrorSummary DOES appear to need t={this.props.t} (so this instance is correct), but other usages of this component are missing it.  Can you check it is passed in to ProcessedErrorSummary everywhere it is required?

Reply:
Actually this doesn't need it, as per explanation for HiddenText component